problem_builder/public/js/review_blocks.js

```diff
@@ -174,7 +174,7 @@ function MentoringTableBlock(runtime, element, initData) {
         })
     });
 
-    function addShare() {
+    function addShare(event) {
```

Comment:
Did you intend to use this `event` parameter, or are you just adding it for consistency?